Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM bigtruedata/sbt
```

Comment:
I can't run docker on my machine to test it but I found nothing obviously wrong 😅